### DISCUSSION TITLE
Add support for reverse proxy mode

### DISCUSSION THREAD

**Discussion by gdubicki** [dotnet/dev-proxy#1225] [Q&A] [answered]
Hi,

This project looks very interesting.

However, my use case is to run it in testing/staging environments, in a k8s cluster to simulate timeouts and other issues with particular services, not to run it locally during development.

Have you considered implementing a reverse proxy mode, where you would provide a target host and port to forward the requests to during startup?

**Comment by waldekmastykarz** [comment, marked as answer]
Hey @gdubicki, thanks for reaching out! Dev Proxy is meant to be used on clients to intercept outgoing API requests and provide you with contextual guidance. We haven't considered implementing routing. The good news is, that Dev Proxy supports plugins so you could [build your own](https://learn.microsoft.com/microsoft-cloud/dev/dev-proxy/how-to/create-custom-plugin). Have you considered using [Yarp](https://microsoft.github.io/reverse-proxy/) for the reverse proxy piece?

**Comment by waldekmastykarz** [comment]
Closing due to lack of further comments
